{{cookiecutter.repostory_name}}/.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Run linter and tests on pull requests
+
+on:
+  pull_request:
+    branches: [master, main]
+
+env:
+  PYTHON_DEFAULT_VERSION: "3.9"
+
+jobs:
+{%- if cookiecutter.ci_use_spellchecker == "y" %}
+  spellchecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Check spelling
+        uses: codespell-project/actions-codespell@2391250ab05295bddd51e36a8c6295edb6343b0e
+        with:
+          ignore_words_list: datas
+{%- endif %}
+{%- if cookiecutter.ci_use_linter == "y" %}
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
+        uses: actions/setup-python@v4
+        with:
+          python-version: {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
+          cache: "pip"
+      - name: Install dependencies
+        run: python -m pip install --upgrade nox pip setuptools
+      - name: Run linters
+        run: nox -vs lint
+{%- endif %}
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
+        uses: actions/setup-python@v4
+        with:
+          python-version: {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
+          cache: "pip"
+      - name: Prepare environment
+        run: ./setup-dev.sh
+      - name: Install dependencies
+        run: python -m pip install --upgrade nox pip setuptools
+      - name: Run dockerized services
+        run: docker-compose up -d
```

Comment:
I assume that's pre-installed in the `ubuntu-latest`, right?

Reply:
https://github.com/kkalinowski-reef/test-single-test-ci/actions/runs/3891510984/jobs/6641822087

It seems so. Worked without issues.